p2p/peering.go

```diff
@@ -119,6 +119,37 @@ type LocalPeer struct {
 	rpMu   sync.Mutex
 }
 
+type SnapShot struct {
```

Comment:
Exported types should have a doc comment describe their purpose and/or usage, and since only one of the fields in this struct changes after handshake, it would be more appropriate to name this RemotePeerInfo.

Reply:
fixed